exec/java-exec/src/main/java/org/apache/drill/exec/planner/FileSystemPartitionDescriptor.java

```diff
@@ -148,13 +139,41 @@ public String getName(int index) {
     return partitionLabel + index;
   }
 
-  private String getBaseTableLocation() {
+  protected String getBaseTableLocation() {
```

Comment:
You are right.  it should remain as private. Originally, I intended to extend this class. But I decided to remove that part of code from this PR. Will update the patch.